src/styles/variables.scss

```diff
@@ -0,0 +1,16 @@
+$calendar-bcg-color: #eee;
+$calendar-gap: 1px;
+$calendar-width: 100px;
+$calendar-height: 100px;
+$calendar-hover-bcg-color: #ffbfcb;
+$start-day: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+$calendar-font: Arial;
```

Comment:
For font variables, it's a good practice to include a fallback font stack. For instance, `$calendar-font: Arial, sans-serif;`. This ensures that if the primary font is unavailable, the browser has alternatives to fall back on.